scripts/create-kubestellar-demo-env.sh

```diff
@@ -103,27 +103,42 @@ cluster_clean_up "kind delete cluster --name kubeflex" &
 cluster_clean_up "kind delete cluster --name cluster1" &
 cluster_clean_up "kind delete cluster --name cluster2" &
 wait
-echo -e "Cluster space clean up has been completed"
+echo -e "\033[33m✔\033[0m Cluster space clean up has been completed"
 
 echo -e "\nStarting context clean up..."
 context_clean_up
-echo "Context space clean up completed"
+echo -e "\033[33m✔\033[0m Context space clean up completed"
 
 echo -e "\nStarting the process to install KubeStellar core: kind-kubeflex..."
 clusters=(cluster1 cluster2)
 for cluster in "${clusters[@]}"; do
    (
      echo -e "Creating cluster ${cluster}..."
      kind create cluster --name "${cluster}" >/dev/null 2>&1 &&
-     kubectl config rename-context "kind-${cluster}" "${cluster}" >/dev/null 2>&1
-     echo -e "${cluster} creation and context setup complete"
+     echo -e "\033[33m✔\033[0m ${cluster} creation and context setup complete"
    ) &
 done
 wait 
 
+for cluster in "${clusters[@]}"; do
+   kubectl config rename-context "kind-${cluster}" "${cluster}" >/dev/null 2>&1
+done
+
+for cluster in "${clusters[@]}"; do
+  if kubectl config get-contexts | grep -w "\b${cluster}\b" >/dev/null 2>&1; then
```

Comment:
I suspect that this test is not doing what you expect:

```console
mspreitz@mjs13 kubestellar % echo kind-cluster1 | grep -w "\bcluster1\b"
kind-cluster1
```

BTW, when I do `man grep` on my Mac, it refers me to `man re_format` for the details on the r.e. language. The latter man page lists `\b` in the "ENHANCED FEATURES" section. There is a plainer way to get the job done, in this particular case.

```console
mspreitz@mjs13 kubestellar % echo " * cluster1 rest" | grep "[ ]cluster1[ ]"
 * cluster1 rest
mspreitz@mjs13 kubestellar % echo " kind-cluster1 rest" | grep "[ ]cluster1[ ]" 
mspreitz@mjs13 kubestellar % 
```